Fsvg.h using deprecated calls.

Building fvwm gives the following depreciating warnings.

```
PictureImageLoader.c:287:9: warning: ‘rsvg_handle_get_dimensions’ is deprecated: Use 'rsvg_handle_get_intrinsic_size_in_pixels' instead [-Wdeprecated-declarations]
  287 |         Frsvg_handle_get_dimensions(rsvg, &dim);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from Fsvg.h:9,
                 from PictureImageLoader.c:46:
/usr/include/librsvg-2.0/librsvg/rsvg.h:708:6: note: declared here
  708 | void rsvg_handle_get_dimensions (RsvgHandle *handle, RsvgDimensionData *dimension_data);
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~
PictureImageLoader.c:360:9: warning: ‘rsvg_handle_render_cairo’ is deprecated: Use 'rsvg_handle_render_document' instead [-Wdeprecated-declarations]
  360 |         Frsvg_handle_render_cairo(rsvg, cr);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/librsvg-2.0/librsvg/rsvg.h:1452:
/usr/include/librsvg-2.0/librsvg/rsvg-cairo.h:88:10: note: declared here
   88 | gboolean rsvg_handle_render_cairo (RsvgHandle *handle, cairo_t *cr);
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
```

This will require a little bit of refactoring how fvwm computes SVG sizes.

--- a/libs/PictureImageLoader.c
+++ b/libs/PictureImageLoader.c
@@ -284,7 +284,23 @@ Bool PImageLoadSvg(FIMAGE_CMD_ARGS)
 	free(allocated_path);
 
 	/* Keep the original aspect ratio when either w or h is 0 */
+#if LIBRSVG_CHECK_VERSION(2,52,0)
+	double		ddw, ddh;
+	RsvgLength	out_w, out_h;
+
+	rsvg_handle_get_intrinsic_size_in_pixels(rsvg, &ddw, &ddh);
+	dim.width = ddw;
+	dim.height = ddh;
+
+	rsvg_handle_get_intrinsic_dimensions(rsvg, NULL, &out_w, NULL, &out_h,
+		NULL, NULL);
+
+	dim.em = out_w.length;
+	dim.ex = out_h.length;
+
+#else
         Frsvg_handle_get_dimensions(rsvg, &dim);
+#endif
 	if (!w && !h)
 	{
 		w = dim.width;
@@ -357,7 +373,19 @@ Bool PImageLoadSvg(FIMAGE_CMD_ARGS)
 	Fcairo_translate(cr, -.5, -.5);
 	Fcairo_scale(cr, 1 / dim.em, 1 / dim.ex);
 
+#if LIBRSVG_CHECK_VERSION(2,52,0)
+	GError *err = NULL;
+	RsvgRectangle vp;
+
+	vp.x = 0;
+	vp.y = 0;
+	vp.width = dim.width;
+	vp.height = dim.height;
+
+	rsvg_handle_render_document(rsvg, cr, &vp, &err);
+#else
 	Frsvg_handle_render_cairo(rsvg, cr);
+#endif
 	Fg_object_unref(FG_OBJECT(rsvg));
 	Fcairo_destroy(cr);
 

